custom/care_pathways/static/care_pathways/ko/report_filter.drilldown_options.js

```diff
@@ -148,9 +148,7 @@ var DrilldownOption = function (select, drilldown_map) {
 };
 
 $.fn.drilldownOptionFilter = function (options) {
-    this.each(function(i) {
-        var viewModel = new DrilldownOptionFilterControl(options);
-        ko.applyBindings(viewModel, $(this).get(i));
-        viewModel.init();
-    });
-};
\ No newline at end of file
+    var viewModel = new DrilldownOptionFilterControl(options);
+    $(this).koApplyBindings(viewModel);
```

Comment:
This is only ever called on an ID, so yeah, length will be 1. Danny & I had a chat about this (don't think i can link to it, but if you expand the outdated diff comments on this PR you'll see it), let me know if that doesn't sound right to you.